src/server/config/config.js

```diff
@@ -5,21 +5,25 @@ config.IS_TESTNET = process.env.IS_TESTNET;
 config.FIRESTORE_USER_ROOT = process.env.FIRESTORE_USER_ROOT;
 config.FIRESTORE_NFT_MINT_SUBSCRIPTION_ROOT =
   process.env.FIRESTORE_NFT_MINT_SUBSCRIPTION_ROOT;
-config.FIRESTORE_WALLET_USER_ROOT = process.env.FIRESTORE_WALLET_USER_ROOT;
+config.FIRESTORE_WALLET_USER_ROOT = 'liker-land-wallet-user-rinkeby';
 
 config.LIKE_CO_CLIENT_ID = process.env.LIKE_CO_CLIENT_ID;
 config.LIKE_CO_CLIENT_SECRET = process.env.LIKE_CO_CLIENT_SECRET;
 
-config.COOKIE_SECRET = process.env.COOKIE_SECRET;
+config.COOKIE_SECRET = process.env.COOKE_SECRET;
 
 config.CRISP_USER_HASH_SECRET = process.env.CRISP_USER_HASH_SECRET;
 
 config.PAYPAL_PDT_HOOK = process.env.PAYPAL_PDT_HOOK;
 config.LIKEPAY_HOOK = process.env.LIKEPAY_HOOK;
 
-config.LIKECOIN_CHAIN_API = process.env.LIKECOIN_CHAIN_API;
-config.LIKECOIN_API_BASE = process.env.LIKECOIN_API_BASE;
-config.EXTERNAL_URL = process.env.EXTERNAL_URL;
+config.LIKECOIN_CHAIN_API = process.env.IS_TESTNET
+  ? 'https://node.testnet.like.co'
+  : 'https://mainnet-node.like.co';
+config.LIKECOIN_API_BASE = process.env.IS_TESTNET
+  ? 'https://api.rinkeby.like.co'
+  : 'https://api.like.co';
+config.EXTERNAL_URL = 'http://localhost:3000';
```

Comment:
Should not include in commit?

Reply:
Magically it did not contain any secret..